### ISSUE TITLE
DeprecationWarning

### ISSUE THREAD

**Issue by mouzzampk2014** [mmgoodnow/cross-seed#180]
HI @mmgoodnow I am seeing these after the latest update (docker)

```
[2/32] Searching for ABC123.2021.2160p.HMAX.WEB-DL.DDP5.1.Atmos.HDR.HEVC-CMRG
(node:1) [https://github.com/node-fetch/node-fetch/issues/1167] DeprecationWarning: form-data doesn't follow the spec and requires special treatment. Use alternative package
(Use `node --trace-deprecation ...` to show where the warning was created)

Found ABC123.2021.2160p.HMAX.WEB-DL.DDP5.1.Atmos.HDR.HEVC-CMRG on tracler1 - injected
```
Thanks

**Comment by mmgoodnow** [comment]
This can safely be ignored. It's specifically an issue with qBittorrent injection.

**Comment by mmgoodnow** [comment]
Fixed by #199

**Comment by mmgoodnow** [comment]
v3.3.1
